packages/RNTester/js/utils/StateManagement.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import {Platform, Linking} from 'react-native';
+import AsyncStorage from '@react-native-community/async-storage';
```

Comment:
This needs to be removed and the new native modules need to be imported here.

Reply:
@manyaagarwal , can't do that for the time being. We will have to create our local state management solution before that which a different issue for this sprint.

Reply:
I am not sure what you mean by local state management solution? From what I understand @chirag-singhal has already implemented the native modules required for async storage in #162

Reply:
@manyaagarwal , I was referring to Chirag's PR only. Will only need to change the import statement

Reply:
Yeah, we can change it after #162 gets merged, but we should do that before merging this PR.